api/app/generated/TrifectalabsRoadqualityV0Client.scala

```diff
@@ -22,16 +22,29 @@ package com.trifectalabs.roadquality.v0.models {
   )
 
   /**
+   * Form the client uses to create a segment rating
+   */
+  case class RatingCreateForm(
```

Comment:
yeah I don't really understand why we need a new rating create form. why can't we use the segment create form?